techMail.js

```diff
@@ -34,11 +34,11 @@ module.exports = function(configuration, templatesDir) {
 
             emailTemplates(templatesDir, function(err, template) {
                 if (err) {
-                    logger.error(err);
+                    logger.error("Unable to configure the template engine... - Details:", err && err.stack || err);
```

Comment:
Not sure I get it... The bug says that the message was `{}`. Is there some cases where we have an error, and cases where we don't ?